RELEASENOTES.md

```diff
@@ -1,5 +1,10 @@
 # cordova-simulate Release Notes
 
+### 0.2.0 (June 9, 2016)
+* Support for additional core plugins
+* Live reload
+* Multiple bug fixes and improvements
```

Comment:
Should we list which new plugins we support?

Should we add "Support for changing the simulated screen size"? I know we implemented it as a "plugin", but I don'T think it fits in the 1st item :)